aladdin.sh

```diff
@@ -284,6 +284,14 @@ function prepare_docker_subcommands() {
     fi
 }
 
+function synchronize_datetime()
+{
+  # When minikube wakes up from sleep, date or time could be out of sync.
+  # Take date + time from host and set it on minikube.
+  echo "Synchronizing date and time on minikube with the host"
+  ssh -i ~/.minikube/machines/minikube/id_rsa docker@$(minikube ip) "docker run --rm --privileged --pid=host alpine nsenter -t 1 -m -u -n -i date -u $(date -u +%m%d%H%M%Y)"
```

Comment:
This may be a silly question, but is there a reason not to just use `minikube ssh`?

Reply:
Could this command be replaced by,
```
minikube ssh "sudo date -s @$(date +%s)"
```
?

Reply:
Yes. Changed it now.